jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/AccountResource.java

```diff
@@ -824,7 +823,7 @@ public PaymentJson apply(final Payment payment) {
         return Response.status(Response.Status.OK).entity(result).build();
     }
 
-    @Timed
+    @TimedResource
```

Comment:
Also, for all these methods `byExternalKey`, you probably want to override the name in `@TimedResource` to combine with the `byId` method (point 2. of https://github.com/killbill/killbill/issues/385).